pokemon-app/src/components/pokedex.tsx

```diff
@@ -0,0 +1,20 @@
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function Pokedex() {
+  return (
+    <View style={styles.container}>
+      <Text>Pokedex component</Text>
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
```

Comment:
You'll see that this file gives you an error at the bottom in github. This is because not having a blank line is kind bad practice for a few different reasons: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

So you might want to just add a blank line at the end of every file 🙂